### DISCUSSION TITLE
File's date in the ZIP install package is incorrect

### DISCUSSION THREAD

**Discussion by DelPoint** [joomla/joomla-cms#37495] [General]
Hi Guys,

I've noticed a small problem (?) regarding file dates. I've found several hundreds of files in the ZIP archive of full/stable install packages, whose date is incorrect, showing a modification date from year 1985.
Affected install packages are:
- **J! 4.0.2**
- **J! 4.1.0**
- **J! 4.1.2**

Funny to observe, that **J! 4.0.0**, and versions from **4.0.3** to **4.0.6** (limits included) are not affected. J! series 3.x not checked.

Just one example: _\media\vendor\bootstrap\css\bootstrap.css_ modification date is set to **26th Oct. 1985**.

I don't know if that can cause any issue anywhere, but it is clear that someone's machine/sw. date settings are incorrect. Should I report this as a bug?

**Comment by HLeithner** [comment]
Hi,

I can't give you a definitive answer on this, I can only tell I built 4.0.0 on Linux, the 4.0.1-4.0.6 are built by the release lead @wilsonge  on macos (my assumption). For 4.1.0 it was build on WSL by release lead @bembelimen .

But what I can say is that it's a problem we need to solve for reproduce able builds and signed updates. We have a sprint in june and will have a look on it.

thanks for your report

**Reply by richard67** [reply]
> We have a spring in june

I assume @HLeithner meant a sprint and not a spring 😉

**Comment by DelPoint** [comment]
Hehe. I wanted to say we'll have a 'summer in June' here, but anyway, go for it :)
Thx. for the reply.
